src/bio/bio_context.c

```diff
@@ -736,7 +736,7 @@ bio_blob_open(struct bio_io_context *ctxt, bool async, bool is_sys, spdk_blob_id
 		 */
 		if (!is_sys || !bio_is_meta_on_ssd_configured()) {
 			rc = smd_pool_get_blob(ctxt->bic_pool_id, xs_ctxt->bxc_tgt_id,
-					       &blob_id);
+					       SMD_DEV_TYPE_DATA, &blob_id);
```

Comment:
Could you fix to pass smd dev type here? extend bio_blob_open

Reply:
Sure, will do that

Reply:
Done